chat-react/src/config.js

```diff
@@ -1,16 +1,16 @@
 export const credentials = {
-  appId: process.env.REACT_APP_CONNECTYCUBE_APP_ID,
-  authKey: process.env.REACT_APP_CONNECTYCUBE_AUTH_KEY,
-  authSecret: process.env.REACT_APP_CONNECTYCUBE_AUTH_SECRET,
+  appId: REPLACE_APP_ID,
+  authKey: 'REPLACE_APP_AUTH_KEY',
+  authSecret: 'REPLACE_APP_AUTH_SECRET',
 };
 
 export const appConfig = {
   debug: {
     mode: 1,
   },
   endpoints: {
```

Comment:
for a sake of simplicity, let's remove endpoints configuration code
Shared users do not need this to change